packages/whats-new/src/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { Guide } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+import { useLocale } from '@automattic/i18n-utils';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import WhatsNewPage from './whats-new-page';
+
+const WhatsNewGuide = () => {
+	const [ showGuide, setShowGuide ] = useState( true );
+	const [ whatsNewData, setWhatsNewData ] = useState( null );
+	const __ = useI18n().__;
+	const locale = useLocale();
+
+	// Load What's New list on first site load
+	useEffect( () => {
+		const proxiedWpcom = wpcom();
+		proxiedWpcom.request = proxyRequest;
+		proxiedWpcom.req
+			.get( { path: `/whats-new/list?locale=${ locale }`, apiNamespace: 'wpcom/v2' } )
+			.then( ( returnedList ) => {
+				setWhatsNewData( returnedList );
+			} );
+	}, [ locale ] );
+
+	const toggleWhatsNew = () => setShowGuide( false );
+
+	if ( ! ( whatsNewData && showGuide ) ) return null;
+
+	return (
+		<Guide
+			// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+			className="whats-new-guide__main"
+			contentLabel={ __( "What's new at WordPress.com", __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'What's New at WordPress.com' )` *ES Score: 12*

Reply:
Seems like we already have a translated version of this, so let's reuse it. Sorry for the back and forth! 

```suggestion
			contentLabel={ __( "What's New at WordPress.com", __i18n_text_domain__ ) }
```